bpf/Makefile

```diff
@@ -135,3 +146,10 @@ $(OUT_PYPERF): $(PYPERF_SRC) $(LIBBPF_HEADERS) $(BPF_HEADERS) | $(OUT_DIR)
 		-O2 -emit-llvm -c -g $< -o $(@:.o=.ll)
 	$(CMD_LLC) -march=bpf -filetype=obj -o $@ $(@:.o=.ll)
 	rm $(@:.o=.ll)
+
+.PHONY: env
+env:
+	$(foreach v,                                        \
+		$(filter-out $(VARS_OLD) VARS_OLD,$(.VARIABLES)), \
+		$(info $(v) = $($(v))))
+	@echo "---"
```

Comment:
I might be missing something, but can't really understand what's happening here?
I think we should put a comment in here, mostly because it's difficult to understand context from just the variable names

Reply:
This is just for debugging. And soon you don't have to deal with makefiles. Stay tuned.